rates/rates.go

```diff
@@ -128,7 +128,7 @@ func (rS *RateS) rateProfileCostForEvent(ctx *context.Context, rtPfl *utils.Rate
 		utils.MetaReq:  args.CGREvent.Event,
 		utils.MetaOpts: args.APIOpts,
 	}
-	var rtIDs utils.StringSet
+	rtIDs := utils.StringSet{}
```

Comment:
Is therre a specific reason for this init here? Thanks!

Reply:
Did this to avoid panic vrom populating that StringSet if  RateIndexedSelect is false